client/jetpack-cloud/sections/partner-portal/issue-multiple-licenses-form/index.tsx

```diff
@@ -115,8 +125,37 @@ export default function IssueMultipleLicensesForm( {
 		?.toString()
 		.split( ',' );
 
+	const handleSelectBundleLicense = useCallback(
+		( product: APIProductFamilyProduct ) => {
+			const productBundle = {
+				...product,
+				quantity,
+			};
+			const index = selectedLicenses.findIndex(
+				( item ) => item.quantity === productBundle.quantity && item.slug === productBundle.slug
+			);
+			if ( index !== -1 ) {
```

Comment:
[Non-blocker] I would encourage to make this a positive condition and make it more easier to reason about.

```
            if ( index === -1 ) {
				// Item doesn't exist, add it
				setSelectedLicenses( [ ...selectedLicenses, productBundle ] );
			} else {
				// Item exists, remove it
				setSelectedLicenses( selectedLicenses.filter( ( _, i ) => i !== index ) );
			}
```